qiita_pet/test/test_portal.py

```diff
@@ -0,0 +1,58 @@
+from unittest import main
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+from mock import Mock
+
+from qiita_db.user import User
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class TestPortal(TestHandlerBase):
+    def test_get(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.get('/admin/portals/studies/')
+        self.assertEqual(response.code, 200)
```

Comment:
This applies to all response.body comments. What should I tests? That the display is accurate all html (too much IMOO, as any minimal change will make the test fail), that the response has the study id/title, other?